translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"same pod running at once.\n"
+"\t\tAlso, if you force delete pods the scheduler may place new pods on those "
+"nodes before the node\n"
+"\t\thas released those resources and causing those pods to be evicted "
+"immediately.\n"
+"\n"
+"\t\tNote that the delete command does NOT do resource version checks, so if "
+"someone\n"
+"\t\tsubmits an update to a resource right when you submit a delete, their "
+"update\n"
+"\t\twill be lost along with the rest of the resource."
+
+#: pkg/kubectl/cmd/stop.go:31
+msgid ""
+"\n"
+"\t\tDeprecated: Gracefully shut down a resource by name or filename.\n"
+"\n"
+"\t\tThe stop command is deprecated, all its functionalities are covered by "
+"delete command.\n"
+"\t\tSee 'kubectl delete --help' for more details.\n"
+"\n"
+"\t\tAttempts to shut down and delete a resource that supports graceful "
+"termination.\n"
+"\t\tIf the resource is scalable it will be scaled to 0 before deletion."
+msgstr ""
+"\n"
+"\t\tDeprecated: Gracefully shut down a resource by name or filename.\n"
+"\n"
+"\t\tThe stop command is deprecated, all its functionalities are covered by "
+"delete command.\n"
+"\t\tSee ‘kubectl delete —help’ for more details.\n"
+"\n"
+"\t\tAttempts to shut down and delete a resource that supports graceful "
+"termination.\n"
+"\t\tIf the resource is scalable it will be scaled to 0 before deletion."
+
+#: pkg/kubectl/cmd/top_node.go:60
+msgid ""
+"\n"
+"\t\tDisplay Resource (CPU/Memory/Storage) usage of nodes.\n"
+"\n"
+"\t\tThe top-node command allows you to see the resource consumption of nodes."
+msgstr ""
+"\n"
+"\t\tMostra os Recursos (CPU/Memory/Storage) utilizados nos nodes.\n"
+"\n"
+"\t\tO comando top-node permite que você veja o consumo de recursos dos nodes."
+
+#: pkg/kubectl/cmd/top_pod.go:53
+msgid ""
+"\n"
+"\t\tDisplay Resource (CPU/Memory/Storage) usage of pods.\n"
+"\n"
+"\t\tThe 'top pod' command allows you to see the resource consumption of "
+"pods.\n"
+"\n"
+"\t\tDue to the metrics pipeline delay, they may be unavailable for a few "
+"minutes\n"
+"\t\tsince pod creation."
+msgstr ""
+"\n"
+"\t\tMostra a utilização de recursos dos pods (CPU/Memory/Storage).\n"
+"\n"
+"\t\tO comando 'top pod' deixa você ver a utilização dos recusrsos dos pods.\n"
+"\n"
+"\t\tDevido ao atraso da pipeline de metrics, o resultado pode estar "
+"indisponível por alguns minutos\n"
+"\t\tdesde a criação do pod."
+
+#: pkg/kubectl/cmd/top.go:33
+msgid ""
+"\n"
+"\t\tDisplay Resource (CPU/Memory/Storage) usage.\n"
+"\n"
+"\t\tThe top command allows you to see the resource consumption for nodes or "
+"pods.\n"
+"\n"
+"\t\tThis command requires Heapster to be correctly configured and working on "
+"the server. "
+msgstr ""
+"\n"
+"\t\tMostra a utilização de recursos (CPU/Memory/Storage).\n"
+"\n"
+"\t\tO comando top deixa você ver a utulização de recursos de nodos e pods.\n"
+"\n"
+"\t\tEste comando necessita que o Heapster esteja corretamente configurado e "
```

Comment:
Heapster has been deprecated in favor of metrics-server, probably this is already updated in the original file

Reply:
still there, maybe in a followup